syntax = "proto3";

// interface de servi√ßo
service Agenda {
  rpc AddContato(Contato) returns(Msg) {}
  rpc RmContato(Contato) returns(Msg) {}
  rpc ListContato(Request) returns(stream Contato){}
}

message Contato {
	string nome = 1;
	string telefone = 2;
}

message Msg {
	string mensagem = 1;
}

message Request{
	string request = 1;
}
